src/app/components/CreateTask.tsx

```diff
@@ -0,0 +1,287 @@
+    name: 'Brroklyn Simmons',
+  },
+]
+
+export const CreateTask = () => {
+  const [displayStatus, setDisplayStatus] = useState(false)
+  const [statusValue, setStatusValue] = useState<string | null>(status[0])
+  const [inputStatusValue, setInputStatusValue] = useState('')
+
+  const setStatusRef = useFocusableInput(displayStatus)
+
+  const [displayAssignee, setDisplayAssignee] = useState(false)
+  const [assigneeValue, setAssigneeValue] = useState<IAssignee>(assignee[0])
+  const [inputAssigneeValue, setInputAssigneeValue] = useState<string>('')
+
+  const setAssigneeRef = useFocusableInput(displayAssignee)
+
+  return (
+    <Box
+      sx={{
+        margin: '0 auto',
+        background: '#ffffff',
+        position: 'absolute',
+        top: '50%',
+        left: '50%',
+        transform: 'translate(-50%, -50%)',
+        boxShadow: '0px 16px 70px 0px rgba(0, 0, 0, 0.5)',
+        border: (theme) => `1px solid ${theme.color.borders.border2}`,
+        borderRadius: '4px',
+        width: '685px',
+      }}
+    >
+      <Stack
+        direction="row"
+        alignItems="center"
+        justifyContent="end"
+        sx={{
+          border: (theme) => `1px solid ${theme.color.borders.borderDisabled}`,
+        }}
+      >
+        <AppMargin size={SizeofAppMargin.MEDIUM} ptb="12px">
+          <Close
+            sx={{ color: (theme) => theme.color.gray[500], cursor: 'pointer' }}
+            onClick={() => store.dispatch(setShowModal())}
+          />
+        </AppMargin>
+      </Stack>
+
+      <AppMargin size={SizeofAppMargin.MEDIUM} ptb="16px">
+        <Stack direction="column" rowGap={1}>
+          <Typography variant="md">Task name</Typography>
+          <StyledTextField type="text" padding="8px 0px" />
+        </Stack>
+        <Stack direction="column" rowGap={1} m="16px 0px">
+          <Typography variant="md">Description</Typography>
+          <StyledTextField
+            type="text"
+            placeholder="Add description..."
+            multiline
+            rows={6}
+            inputProps={{ style: { resize: 'vertical' } }}
+          />
+        </Stack>
+
+        <Stack direction="row" columnGap={3} position="relative">
+          <Stack direction="column">
+            <SecondaryBtn
+              startIcon={statusIcons[statusValue || 'Todo']}
+              buttonContent={
+                <Typography variant="bodySm" lineHeight="16px" sx={{ color: (theme) => theme.color.gray[600] }}>
+                  {statusValue}
+                </Typography>
+              }
+              handleClick={() => {
+                setDisplayAssignee(false)
+                setDisplayStatus((prev) => !prev)
+              }}
+            />
+            <Box
+              sx={{
+                position: 'absolute',
+                top: 35,
+                width: '180px',
+                display: displayStatus ? 'block' : 'none',
+              }}
+            >
+              <StyledAutocomplete
```

Comment:
Could we modularize this to a separate component? Even though we will not reuse this, it might make the code more maintainable